src/components/Offramp/Confirm.view.tsx

```
@@ -441,9 +600,40 @@ export const ConfirmCashoutView = ({
+                                    offrampForm?.recipient
+                                }
                             </span>
                         </div>
+
+                        {offrampType == OfframpType.CLAIM && (
+                            <div className="flex w-full flex-row items-center justify-between gap-1 px-2 text-h8 text-gray-1">
+                                <div className="flex w-max  flex-row items-center justify-center gap-1">
+                                    <Icon name={'forward'} className="h-4 fill-gray-1" />
+                                    <label className="font-bold">Route</label>
+                                </div>
+                                {false ? (
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)